src/OrchardCore.Modules/OrchardCore.Media/Views/MediaCache/Index.cshtml

```diff
@@ -12,7 +12,7 @@
             <li class="list-group-item">
                 <div class="properties">
                     <div class="related">
-                        <a asp-action="Purge" class="btn btn-sm btn-danger float-right" data-title="@T["Purge All"]" data-message="@T["Are you sure you want to purge the entire asset cache?"]" itemprop="RemoveUrl UnsafeUrl">@T["Purge All"]</a>
+                        <a asp-action="Purge" class="btn btn-sm btn-danger float-right" data-title="@T["Purge All"]" data-message="@T["Are you sure you want to purge the entire asset cache?"]" data-ok-class="btn-danger" itemprop="RemoveUrl UnsafeUrl">@T["Purge All"]</a>
```

Comment:
btn-danger is the default value, I though you may using warning

Reply:
Ah, if it is the default, I'll remove it then. Because red for danger when deleting things is consistent with the rest of the ui